dts/arm/st/l4/stm32l471.dtsi

```diff
@@ -282,3 +282,25 @@
 		status = "disabled";
 	};
 };
+
+&pwr {
+	wkup-pin@1 {
+		wkup-gpios = <&gpioa 0 STM32_PWR_WKUP_PIN_SRC_0>;
+	};
+
+	wkup-pin@2 {
+		wkup-gpios = <&gpioc 13 STM32_PWR_WKUP_PIN_SRC_0>;
+	};
+
+	wkup-pin@3 {
+		wkup-gpios = <&gpioe 6 STM32_PWR_WKUP_PIN_SRC_0>;
+	};
+
+	wkup-pin@4 {
+		wkup-gpios = <&gpioa 2 STM32_PWR_WKUP_PIN_SRC_0>;
+	};
+
+	wkup-pin@5 {
+		wkup-gpios = <&gpioc 5 STM32_PWR_WKUP_PIN_SRC_0>;
+	};
+};
```

Comment:
Same list is present in `dts/arm/st/l4/stm32l4r5Xi.dtsi`, and after a quick look to https://github.com/STMicroelectronics/STM32_open_pin_data, it seems that those pins are the always the same on all SoC of L4 series, the only gotcha is that on some variants (small variants or packages) some pins are not present.
For instance wkup-pin 3 not available in STM32L471R(E-G).

So my proposal would be to put the whole list in https://github.com/zephyrproject-rtos/zephyr/blame/main/dts/arm/st/l4/stm32l4.dtsi#L481-L482
and then delete nodes if needed in impacted packages.

For instance:
```
&pwr {
	/delete-node/ wkup-pin@3;
};
```

Note: it might require to add nodelabels to allow the use of /delete-node/ syntax:
```
	wkuppin4: wkup-pin@4 { ....}
[...]
&pwr {
	/delete-node/ wkuppin4;
};
```

Reply:
Yes, the stm32L4 serie has 5 WKUP pins, same for the stm32L47x/8x/9x/ax and stl32L4plus serie, 
but the gpio associated with each wakeup pin (example GPIOC 13 for the wakeUp pin #2) is defined by each stm32L4 datasheet.